ignite/templates/typed/map/files/tests/messages/x/{{moduleName}}/keeper/msg_server_{{typeName}}_test.go.plush

```diff
@@ -16,9 +16,11 @@ import (
 var _ = strconv.IntSize
 
 func Test<%= TypeName.UpperCamel %>MsgServerCreate(t *testing.T) {
-	k, ctx := keepertest.<%= title(ModuleName) %>Keeper(t)
+	k, ctx, addressCodec := keepertest.<%= title(ModuleName) %>Keeper(t)
 	srv := keeper.NewMsgServerImpl(k)
-	<%= MsgSigner.LowerCamel %> := "A"
+	<%= MsgSigner.LowerCamel %>, err := addressCodec.BytesToString([]byte("signerAddr__________________"))
```

Comment:
Is there any reason for the underliner in the `signerAddr__________________`?

Reply:
Just took that from what we do on the SDK. No specific reason.